modules/my_health/app/controllers/my_health/v1/vitals_controller.rb

```diff
@@ -4,7 +4,12 @@ module MyHealth
   module V1
     class VitalsController < MrController
       def index
-        resource = client.list_vitals
+        resource = if params[:from] && params[:to]
+                     client.list_vitals(params[:from], params[:to])
+                   else
+                     client.list_vitals
```

Comment:
list_vitals requires two arguments.

Reply:
@RachalCassity it only requires arguments when using the Lighthouse Client. Definitely open to a better way of doing this, maybe it should be conditioned on the client class name?

Reply:
🤔 or maybe we should make the from and to params optional, the [LH API does not require them](https://developer.va.gov/explore/api/patient-health/docs?version=current)

Reply:
`def list_vitals(from_date: nil, to_date: nil)`